File: programers/134240.py
# 푸드 파이트 대회
def solution(food):
    prestr = ""
    for i in range(1, len(food)):
        tmp = food[i]//2
        while tmp:
            prestr += str(i)
            tmp -= 1
    answer = [prestr, prestr[::-1]]
    return "0".join(answer)

File: programers/java/134240.java
import java.util.*;
class Solution {
    //푸드 파이트 대회

    public String solution(int[] food) {
        StringBuilder strbuilder=new StringBuilder();
        for(int i=1;i<food.length;i++){
            int tmp=food[i]/2;
            while (tmp>0){
                strbuilder.append(i);
                tmp--;
            }
        }
        
        List<String> answer=new ArrayList<>();
        answer.add(strbuilder.toString());
        answer.add(reverse(strbuilder));
        
        return String.join("0",answer);        
    }
    public String reverse(StringBuilder str){
        return new StringBuilder(str).reverse().toString();
    }
}